### DISCUSSION TITLE
SEARCH VALUE ENTERED IN FORM IN ATTRIBUTE TABLE

### DISCUSSION THREAD

**Discussion by CACG1022** [opengisch/QField#3993] [Q&A]
Hello, I am counting a value entered by form; In qgis it works fine and it finds the searched value, but in the Qfield App it doesn't work. Next I relate the formula that I am using in Qgis

FUNCTION:
aggregate(layer:='HILOS',aggregate:='count',
expression:="NOMBRE",filter:="BUFER"=attribute(@parent,'PUERTOSPLI'))
![PREGUNTAQFIELD](https://user-images.githubusercontent.com/118937011/222545614-8cf18697-5a5e-49d5-a13c-06f750f3edb7.png)
![PREGUNTAQFIELD](https://user-images.githubusercontent.com/118937011/222545651-c9148e6c-7148-47a1-a777-f0d6d95dbd08.png)

**Comment by nirvn** [comment]
@CACG1022 , could you attach a test project please?

**Reply by CACG1022** [reply]
[PROYECTO1.zip](https://github.com/opengisch/QField/files/10882936/PROYECTO1.zip)

Hello. attached the project

**Reply by nirvn** [reply]
@CACG1022 , thanks, this was an issue we were not aware of, it'll be fixed in QField 2.7.

**Reply by CACG1022** [reply]
OK, thank you very much, when will version 2.7 come out?

**Comment by alexander131290** [comment]
¡Hola!

 He experimentado el mismo incoveniente con algunas funcinoes de agregación, y específicamente con la función "count". Todo funciona correctamente en QGIS, peron no en QField.

https://github.com/opengisch/QField/discussions/4101
